pages/addData.js

```diff
@@ -0,0 +1,38 @@
+import { useRouter } from "next/router";
+import FormDB from "../components/forms/FormDB";
+import { getAllDays } from "../services/dayService";
+import { CardContainer } from "../components/CardContainer";
+
+export async function getServerSideProps() {
+  const days = await getAllDays();
+
+  return {
+    props: {
+      days,
+    },
+  };
+}
+
+export default function AddData({ days }) {
+  const router = useRouter();
+
+  async function handleSubmit(data) {
+    try {
+      const response = await fetch("/api/Days", {
+        method: "POST",
+        body: JSON.stringify(data),
+      });
+      const result = await response.json();
```

Comment:
why didnt u use the results returning from the API? The server side redering getServerSideProps() is not neccessary anymore. But maybe it was from ur previous implementation right?

Reply:
I think I need them to put them into "CardContainer" in line 35?!

Reply:
no the data will come from result respone.json() at line 25. U can store this result in a state variable then pass this state variable to line 35

Reply:
Done! Works perfectly, thank you!